packages/ezflow_extension/ezextension/ezflow/design/standard/templates/page/zone.tpl

```diff
@@ -1,7 +1,15 @@
 <div class="object-left">
     <select name="ContentObjectAttribute_ezpage_block_type_{$attribute.id}_{$zone_id}">
+    {def $allowed_zones = false()}
     {foreach ezini( 'General', 'AllowedTypes', 'block.ini' ) as $type}
-        <option value="{$type}">{ezini( $type, 'Name', 'block.ini' )}</option>
+        {if ezini_hasvariable( $type, 'AllowedZones', 'block.ini' )}
```

Comment:
Isn't it more logical to configure which blocks can go into which zones in `zone.ini` with the following config:

[2ZonesLayout1]
ZoneTypeName=2 zones (layout 1)
Zones[]=left
Zones[]=right
AllowedBlocks_left[]
AllowedBlocks_left[]=ContentGrid

The way it is currently defined, when you say that block can go into a zone with `right` identifier, what does it mean exactly? `right` identifier doesn't mean anything by itself without context of a layout.